server/game/GameActions/ChooseGameAction.js

```diff
@@ -45,8 +45,10 @@ class ChooseGameAction extends GameAction {
         super.preEventHandler(context);
         let activePromptTitle = this.activePromptTitle;
         let choices = Object.keys(this.choices);
-        choices = choices.filter((key) =>
-            this.choices[key].some((action) => action.hasLegalTarget(context))
+        choices = choices.filter(
+            (key) =>
+                this.choices[key].some((action) => action.hasLegalTarget(context)) ||
+                this.choices[key].length === 0
```

Comment:
this could have implications... I will analyze more carefully.

Reply:
This seemed safe since we don't use `chooseAction` anywhere else as far as I can tell.  The other option I thought of was making a `NullAction`/`NoopAction` but that seemed lame.

Reply:
I did not understand why you had to add this check.

Reply:
@romanogit it is to allow choices that don't have any actions in their handler.  I couldn't find any other example of a choice that did nothing when you clicked on it.  Perhaps there was a better way to implement an action that allows a choice like that?